[2016-03-24 20:33:37] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'DB' not found' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php:18
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-24 20:34:15] lumen.ERROR: exception 'PDOException' with message 'could not find driver' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('sqlsrv:Server=1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('sqlsrv:Server=1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 20:35:09] lumen.ERROR: exception 'PDOException' with message 'could not find driver' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('sqlsrv:Server=1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('sqlsrv:Server=1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 20:38:14] lumen.ERROR: exception 'ErrorException' with message 'The use statement with non-compound name 'DB' has no effect' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php:14
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/bootstrap/app.php(97): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(2, 'The use stateme...', '/Users/rodrigop...', 14, Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/bootstrap/app.php(97): {closure}()
#2 [internal function]: {closure}(Object(Laravel\Lumen\Application))
#3 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(92): call_user_func(Object(Closure), Object(Laravel\Lumen\Application))
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/bootstrap/app.php(98): Laravel\Lumen\Application->group(Array, Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(14): require('/Users/rodrigop...')
#6 {main}  
[2016-03-24 20:38:38] lumen.ERROR: exception 'PDOException' with message 'could not find driver' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('sqlsrv:Server=1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('sqlsrv:Server=1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 20:40:03] lumen.ERROR: exception 'PDOException' with message 'could not find driver' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('sqlsrv:Server=1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('sqlsrv:Server=1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 20:40:07] lumen.ERROR: exception 'PDOException' with message 'could not find driver' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('sqlsrv:Server=1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('sqlsrv:Server=1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 20:53:35] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 20:58:49] lumen.ERROR: exception 'ErrorException' with message 'mssql_connect(): Unable to connect to server: 10.10.1.174:65335' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php:21
Stack trace:
#0 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(2, 'mssql_connect()...', '/Users/rodrigop...', 21, Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(21): mssql_connect('10.10.1.174:653...', 'CCCOM\\sgoulet', 'cccsgoulet')
#2 [internal function]: Closure->{closure}()
#3 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#8 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#11 {main}  
[2016-03-24 21:00:04] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-24 21:32:23] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 /Users/rodrigopassos/Documents/Dative/ClientRepos/CCC/FinancialModelAPI/public/index.php(28): Laravel\Lumen\Application->run()
#21 {main}  
[2016-03-25 17:21:23] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:13:36] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(18): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:24:44] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:42:56] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:42:58] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:43:45] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:43:47] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:45:00] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:45:19] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:46:57] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:46:59] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:47:16] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:47:23] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
[2016-03-25 19:47:41] lumen.ERROR: exception 'ErrorException' with message 'Undefined variable: cccsgoulet' in /var/www/fim-api/app/Http/routes.php:36
Stack trace:
#0 /var/www/fim-api/app/Http/routes.php(36): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined varia...', '/var/www/fim-ap...', 36, Array)
#1 [internal function]: Closure->{closure}()
#2 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#3 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#4 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#5 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#6 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#7 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#8 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#9 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#10 {main}  
[2016-03-25 19:52:03] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] Unable to connect: Adaptive Server is unavailable or does not exist (severity 9)' in /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php:55
Stack trace:
#0 /var/www/fim-api/vendor/illuminate/database/Connectors/Connector.php(55): PDO->__construct('dblib:host=10.1...', 'CCCOM\\sgoulet', 'cccsgoulet', Array)
#1 /var/www/fim-api/vendor/illuminate/database/Connectors/SqlServerConnector.php(32): Illuminate\Database\Connectors\Connector->createConnection('dblib:host=10.1...', Array, Array)
#2 /var/www/fim-api/vendor/illuminate/database/Connectors/ConnectionFactory.php(61): Illuminate\Database\Connectors\SqlServerConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 /var/www/fim-api/vendor/illuminate/database/Connection.php(864): call_user_func(Object(Closure))
#5 /var/www/fim-api/vendor/illuminate/database/Connection.php(732): Illuminate\Database\Connection->getPdo()
#6 /var/www/fim-api/vendor/illuminate/database/Connection.php(621): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 /var/www/fim-api/vendor/illuminate/database/Connection.php(342): Illuminate\Database\Connection->run('select * from [...', Array, Object(Closure))
#8 [internal function]: Illuminate\Database\Connection->select('select * from [...')
#9 /var/www/fim-api/vendor/illuminate/database/DatabaseManager.php(317): call_user_func_array(Array, Array)
#10 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->__call('select', Array)
#11 /var/www/fim-api/app/Http/routes.php(35): Illuminate\Database\DatabaseManager->select('select * from [...')
#12 [internal function]: Closure->{closure}()
#13 /var/www/fim-api/vendor/illuminate/container/Container.php(507): call_user_func_array(Object(Closure), Array)
#14 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(500): Illuminate\Container\Container->call(Object(Closure), Array)
#15 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 /var/www/fim-api/vendor/laravel/lumen-framework/src/Concerns/RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(Object(Illuminate\Http\Request))
#20 /var/www/fim-api/public/index.php(28): Laravel\Lumen\Application->run(Object(Illuminate\Http\Request))
#21 {main}  
